Automation/SQL/azureDeploy.ps1

```diff
@@ -0,0 +1,24 @@
+param(
+        [string] [Parameter(Mandatory=$true)] $location,
+        [string] [Parameter(Mandatory=$true)] $resourceGroupName,
+        [string] [Parameter(Mandatory=$true)] $serverName,
+        [string] [Parameter(Mandatory=$true)] $tenantId,          
+        [string] [Parameter(Mandatory=$true)] $applicationId,
+        [string] [Parameter(Mandatory=$true)] $secret,
+        [string] [Parameter(Mandatory=$true)] $subscriptionId
+)
+
+$pathToAzSqlTemplate = "./azureDeploy.json"
+   
+& "../login.ps1" $tenantId $applicationId $secret $subscriptionId
+
+New-AzResourceGroup -Name $resourceGroupName -Location "$location"
+
+Write-host "Creating primary server..."
+New-AzSqlServer -ResourceGroupName $resourceGroupName `
```

Comment:
make sure to add the admin username and password as input parameters so they don't need to be added in a separate step

Reply:
Done